app/assets/stylesheets/modern-business.scss

```diff
@@ -90,4 +90,13 @@ footer {
     header.carousel .carousel {
         height: 70%;
     }
-}
\ No newline at end of file
+}
+
+.carousel-caption h2 {
+  font-size: 5em;
+}
+
```

Comment:
Avoid using id selectors<br>Selector `myCarousel` should be written in lowercase with hyphens